G/GMP/GMP@6.2.1/patches/CVE-2021-43618.patch

```diff
@@ -0,0 +1,24 @@
+# Origin: https://gmplib.org/repo/gmp-6.2/rev/561a9c25298e
```

Comment:
Note that the patch should go in the bundled directory. This isn't being picked up actually

Reply:
Ah forgot that.